app/src/main/java/com/example/javaopencommerce/item/dtos/ItemDto.java

```diff
@@ -0,0 +1,21 @@
+package com.example.javaopencommerce.item.dtos;
+
+import com.example.javaopencommerce.image.ImageDto;
+import java.math.BigDecimal;
+import lombok.AllArgsConstructor;
+import lombok.Builder;
+import lombok.Data;
+
+@Data
+@Builder
+@AllArgsConstructor
```

Comment:
I don't think you need `@AllArgsConstructor` with `@Data`.

Reply:
done!